.github/workflows/trigger-repository-archival.yaml

```
@@ -0,0 +1,48 @@
+name: Trigger Unowned Repository Scan
+
+on:
+  schedule:
+    - cron: "0 9 * * *"
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: Whether to dry-run archival
+        required: true
+        default: "true"
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  trigger:
+    name: Trigger Unowned Repository Scan
+    runs-on: [self-hosted, universal]
+    steps:
+      - name: Send repository_dispatch event to GitHub
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const [owner, repository] = '${{ github.repository }}'.split("/")
+            if (!owner || !repository) {
+              core.error('No owner or repository was found in the runner context - exiting.')
+              process.exit(1)
+            }
+
+            const endpoint = `/repos/${owner}/${repository}/dispatches`
+            core.info(`Dispatching 'repository_dispatch' event to '${endpoint}'`)
+
+            const dryRunInput = ${{ github.event.inputs.dry-run }}
+            // Dry-run if the input is not present or set to 'true'. Otherwise disable dry-run.
+            const shouldDryRun = dryRunInput === undefined || dryRunInput === '' ? true : dryRunInput === 'true'
```

```suggestion
            const shouldDryRun = dryRunInput !== 'false'
```

This would mean we always dry run _unless_ we specifically mark dry run as false.